whats_fresh/whats_fresh_api/templates/product.html

```diff
@@ -97,15 +103,64 @@ <h4 id='error_tag' class='light warning'></h4>
 {% endblock content %}
 {% block footer %}
 <script>
+
     var preparation_options = {{ json_preparations|safe }};
 
     number_of_preparations = 0;
 
+    function popitup(url) {
+      $.ajax({
+        type: "GET",
+        url: url,
+        headers: { 'X-CSRFToken': '{{ csrf_token }}' },
+        success: function(data) {
+
+          $("#dialog-form").html(data).dialog({modal:true}, {title: "New Preparation"}, {width: 700}, {height: 700}, {draggable: false}).dialog('open');
+        }
+      });
+      return false;
+    }
+
+
+    $("#dialog-form").on('submit', function (e) {
+
+         e.preventDefault();
+
+        $.ajax({
+            type: "POST",
+            url: '/entry/products/new/preparations/new',
+            data: $("#popup-form").serialize(),
+            headers: { 'X-CSRFToken': '{{ csrf_token }}' },
+            success: function(data)
+            {
+                if (data.constructor === "test".constructor) {       // checks if the data is a string
+                    $("#dialog-form").html(data);
+                    return
+                }
+
+                alert("New Preparation saved.");
```

Comment:
@tschuy Do we want to get rid of this and similar alerts ( New Product Saved etc.. ) ?